packages/form-participation/src/validators.ts

```diff
@@ -0,0 +1,75 @@
+import { Validator } from './index';
+
+export interface RequiredHost extends HTMLElement {
+  required: boolean;
+};
+
+export const requiredValidator: Validator = {
+  attribute: 'required',
+  key: 'valueMissing',
+  message: 'You must include a value',
```

Comment:
nit: if this is going to be the default required message, maybe we should use a friendlier tone? or something less personal like not using "you must" but having messaging like in the minLengthValidator like `Value must not be empty`

Reply:
That's a fair point, I think if this is something open-wc would like to continue with, these should all be massaged a bit more (and maybe a more definitive list of validators created). Happy to revisit all of these.

Reply:
I think the bigger question is whether customization of messages here is simple. Anywhere that text strings are delivered to a user we need to keep in mind the need to translate them, etc.

Reply:
There are a number of strategies built in currently to modify error messages including an instance callback that passes the validation key to the user and expects a returned string. That callback could obviously do anything.